mongo/options/gridfsoptions.go

```diff
@@ -210,6 +210,10 @@ func MergeNameOptions(opts ...*NameOptions) *NameOptions {
 
 // GridFSFindOptions represents options that can be used to configure a GridFS Find operation.
 type GridFSFindOptions struct {
+	// If true, the server can write temporary data to disk while executing the find operation. The default value
```

Comment:
This should have the same note about server versions as `FindOptions`.

Reply:
done